libopenage/util/fds.h

```diff
@@ -85,7 +85,7 @@ class FD {
 	void restoreinputmode();
 
 	#ifndef _WIN32
-	struct termios old_tio;
+		struct termios old_tio;
```

Comment:
the indent of `old_tio` is right, but you can move the `#ifndef` to the beginning of the line.